### DISCUSSION TITLE
MOT2020 Labels cannot be downloaded

### DISCUSSION THREAD

**Discussion by QiyuLuo** [bdd100k/bdd100k#234] [Q&A] [answered]
When I clicked MOT2020 Labels, the following error appeared on the web page. Although Xunlei caught the download link, it could not be linked and the download failed.
```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Request has expired</Message>
<X-Amz-Expires>2</X-Amz-Expires>
<Expires>2022-05-04T01:52:05Z</Expires>
<ServerTime>2022-05-04T01:52:06Z</ServerTime>
<RequestId>PD1XEPYTDH1H6GG9</RequestId>
<HostId>wq0R8DZXvGPtD9pevQTGYGhYXJepK5S8nXZ4zI5/XzhjviukZkHuiyD0MJgvm2PP1kRoEYtaUj4=</HostId>
</Error>
```

**Comment by QiyuLuo** [comment]
100K Images and Detection 2020 Labels have the same problem.

**Comment by tobiasfshr** [comment, marked as answer]
Please follow the [downloading instructions](https://doc.bdd100k.com/download.html), which is the only way to download the data due to license restrictions. Direct link downloading will meet access issues.

**Reply by QiyuLuo** [reply]
I tried to download the data, but the download failed.
![QQ截图20220504164101](https://user-images.githubusercontent.com/34224700/166649777-cc8cd84a-cfab-4573-bcee-7c54ba266cd5.png)
![image](https://user-images.githubusercontent.com/34224700/166649876-96b1ab35-7471-44c9-819e-e208e70f3459.png)

**Reply by tobiasfshr** [reply]
okay but i cannot reproduce this problem on my machine. So try a different machine / browser / even VPN maybe?

**Reply by QiyuLuo** [reply]
> 

Maybe this page will not appear if you click on it, you can try to download MOT2020 Labels and see if it can be downloaded successfully?

**Reply by tobiasfshr** [reply]
i can download the MOT 20 label successfully

**Reply by QiyuLuo** [reply]
I tried several computers and mobile phones, but they failed to download. Could you please send MOT2020 Labels to my email? 
My email is 1327387753@qq.com. Thank you for your help.

**Reply by simomagi** [reply]
I have the same problem with the label download; I am able to correctly download the videos but the labels seem not available.

**Reply by tobiasfshr** [reply]
We tried it with multiple, devices, browsers, operating systems and even a different country, and we can all download the data fine. I think this might be a region based problem, so i'd suggest you try to use a VPN for downloading. We _cannot_ change anything to the downloading procedure due to license requirements.

**Reply by ayang-1434** [reply]
> We tried it with multiple, devices, browsers, operating systems and even a different country, and we can all download the data fine. I think this might be a region based problem, so i'd suggest you try to use a VPN for downloading. We _cannot_ change anything to the downloading procedure due to license requirements.

I also meet this problem and I have tried on serveal computers ,besides I also use vpn, but it made no use. Could you please send MOT2020 Lables and 100K images send to my email? My email is zhangquan1434@126.com. Thank you so much !

**Reply by elhamaahmadi** [reply]
BDD100K dataset cannot be downloaded, I also meet this problem and I have tried on serveal computers ,besides I also use vpn, but it made no use. please help me, Could you please send MOT2020 Lables and 100K images send to my email? My email is zahra.nazemian@yahoo.com Thank you so much.

**Reply by teodoraspasojevic** [reply]
Hello, did anyone receive the MOT2020 Labels?

**Reply by teodoraspasojevic** [reply]
Hi everyone, I have found the labels here: https://dl.cv.ethz.ch/bdd100k/data/

**Reply by QQQ6666qqq** [reply]
@teodoraspasojevic Hello,I want to download the labels of bdd100k mots. According to the format in the picture, I want to know which one I should download.Thank you so much.
<img width="774" height="1147" alt="image" src="https://github.com/user-attachments/assets/361f401c-ee50-4044-a8f9-4b8310df911d" />

**Reply by Runfree-Coco** [reply]
I've been trying to download the BDD100K MOT2020 Labels (box_track_20) for weeks. Tried official tools, OpenDataLab, OpenXLab, VPN, and the ETH mirror link above — nothing works.Would anyone who has the labels (~2.5GB, contains box_track_20/val with 200 JSONs) be willing to share?My email: 2326942199@qq.com,Thanks a lot!

**Reply by HyungseopLee** [reply]
I've been trying to download the BDD100K MOT2020 Labels (box_track_20) for weeks. Tried official tools, OpenDataLab, OpenXLab, VPN, and the ETH mirror link above — nothing works.Would anyone who has the labels (~2.5GB, contains box_track_20/val with 200 JSONs) be willing to share?My email: [hhss0927@inu.ac.kr](mailto:hhss0927@inu.ac.kr),Thanks a lot!

**Reply by thanhhvnqb** [reply]
I've been trying to download the BDD100K MOT2020 Labels (box_track_20) for weeks. Tried official tools, OpenDataLab, OpenXLab, VPN, and the ETH mirror link above — nothing works. Would anyone who has the labels (~2.5GB, contains box_track_20/val with 200 JSONs) be willing to share?My email: [thanhhv@qbu.edu.vn](mailto:thanhhv@qbu.edu.vn),Thanks a lot!
